### DISCUSSION TITLE
Watch mode for dependencies

### DISCUSSION THREAD

**Discussion by ocavue** [vercel/turborepo#206] [Ideas]
Hi. I'm wondering how to *watch* the dependencies when using turbo repo. There is an article from RushStack that explains this issue with more details as well as the solution used by Rush: https://rushjs.io/pages/advanced/watch_mode/.

**Comment by alexn-s** [comment]
i am asking myself too. probably a watch flag or integration with something like nodemon would be appropriate.

**Comment by anthonyshew** [comment]
Follow along with #986.
